client/src/main/java/org/devfleet/esi/api/service/ESIServiceFactory.java

```diff
@@ -0,0 +1,31 @@
+package org.devfleet.esi.api.service;
```

Comment:
So I was mentioning a facade in a previous comment. 
This interface definitely is one and it's great. Should it be simply called ESI?